src/backends/torch/torchlib.cc

```diff
@@ -531,9 +531,8 @@ namespace dd
     // create solver
     tsolver.create(_module);
 
-    int it = 0;
-    // reload solver and set it value accordingly
-    it = tsolver.load(this->_mlmodel._sstate, _main_device);
+    int it = tsolver.resume(ad_mllib, this->_mlmodel, _main_device,
```

Comment:
Need to reload `best_to_remove` as well ?

Reply:
Looking at the code it seems that `best_to_remove` is always equal to -1

Reply:
> Need to reload `best_to_remove` as well ?

nice one, I did not remember the existence of this one

Reply:
> Looking at the code it seems that `best_to_remove` is always equal to -1

It should change, it is updated by save_if_best (changed if best)

Reply:
ok I see